gnosis/safe-react
# style: Outlined input migration

## What it solves
Resolves #3475 #534 

## How this PR fixes it
All input/form fields now use the outlined style instead of filled.

## How to test it

1. Open the Safe app
2. Interact with every form
3. Observe that input fields are outlined instead of filled as specified [here](https://www.figma.com/file/uqDJqIGikIB1zVs56XKnIu/Design-System---Web?node-id=476%3A4268)

Additionally, forms should be tested on Windows to prevent a potential regression with overflowing content within input fields.

## Screenshots
![Screenshot 2022-02-23 at 18 20 02](https://user-images.githubusercontent.com/5880855/155372322-fb20e55b-eae8-4fd6-a1d8-820c538508e5.png)
